File: python/6_kyu/your_order_please.py
# Your Order, Please
# https://www.codewars.com/kata/55c45be3b2079eccff00010f/train/python

def order(sentence):
    sentence = sentence.split(" ")
    ans = []
    
    for i in range(1, len(sentence) + 1):
        for j in sentence:
            if str(i) in j:
                ans.append(j)
    
    return " ".join(ans)


print(order("4of Fo1r pe6ople g3ood th5e the2"))

File: javascript/6_kyu/your_order_please.js
// Your Order, Please
// https://www.codewars.com/kata/55c45be3b2079eccff00010f/train/javascript

function order(words) {
    words = words.split(" ");
    var ans = [];

    for (var i = 1; i < words.length + 1; i++) {
        for (var j = 0; j < words.length; j++) {
            if ( words[j].includes(i.toString()) ) {
                ans.push(words[j]);
            }
        }
    }

    return ans.join(" ");
}

console.log(order("4of Fo1r pe6ople g3ood th5e the2"));